client/trackpack/src/components/homepage.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+import Topnav from "./Topnav";
+import "./homepage.css";
+import { Flex, Box } from "@chakra-ui/react";
+class Homepage extends Component {
+  state = {
+    about: "",
+  };
+
+  renderAboutUs() {
+    if (this.state.about === "signIn") return null;
+  }
+  render() {
+    return (
+      <Flex
```

Comment:
Flex is a chakra component. Does it work with without the ChakraProvider in App.js?